lib/lotus/helpers/html_helper/empty_html_node.rb

```diff
@@ -45,13 +59,31 @@ def to_s
         # @api private
         def attributes
           return if @attributes.nil?
-          result = [nil]
+          result = ''
 
-          @attributes.each do |name, value|
-            result << %(#{ name }="#{ value }")
+          @attributes.each do |attribute_name, value|
+            if boolean_attribute?(attribute_name)
+              result << boolean_attribute(attribute_name, value)
+            else
+              result << attribute(attribute_name, value)
+            end
           end
 
-          result.join(ATTRIBUTES_SEPARATOR)
+          result
+        end
+
+        def boolean_attribute?(attribute_name)
+          BOOLEAN_ATTRIBUTES.include?(attribute_name.to_s)
+        end
+
+        # Do not render boolean attributes when their value is _false_.
+        def boolean_attribute(attribute_name, value)
+          return '' if !value
```

Comment:
what if users provide value as text, such as `true`? What do you think about checking this value against `true` instead? Furthermore, I like to see this `if` inside `boolean_attribute?`